djangocms_versioning_filer/helpers.py

```diff
@@ -95,10 +93,12 @@ def filename_exists(request, folder_id=None):
             upload = list(request.FILES.values())[0]
             filename = upload.name
         else:
-            filename = request.GET.get('qqfile', False) or request.GET.get('filename', False) or ''
+            filename = request.POST.get('file')
         if File._original_manager.filter(
             original_filename=filename,
             folder_id=folder_id
         ):
+            FILE_EXISTS = (_('The file %s already exists, do you want to overwrite this?') % filename)
```

Comment:
Using a constant now the value isn't defined high up is strange, this can be a normal variable now. Also for translation strings django recommends using a placeholder so that the string can be translated without the variable filename:

```suggestion
            file_exists_message = _('The file %(filename)s already exists, do you want to overwrite this?') % { 'filename': filename }
```

From the django docs: https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#standard-translation
(The caveat with using variables or computed values, as in the previous two examples, is that Django’s translation-string-detecting utility, [django-admin makemessages](https://docs.djangoproject.com/en/4.0/ref/django-admin/#django-admin-makemessages), won’t be able to find these strings. More on makemessages later.)